Livecheckables/htpdate.rb

```diff
@@ -1,4 +1,4 @@
 class Htpdate
-  livecheck :url   => "http://www.vervest.org/htp/download",
-            :regex => %r{href=.*?/htpdate-([0-9\.]+)\.t}
+  livecheck :url   => "http://www.vervest.org/htp/archive/c/?C=M&O=D",
+            :regex => %r{href="htpdate-(\d+(?:\.\d+)+)\.t}
```

Comment:
```suggestion
            :regex => /href="htpdate-(\d+(?:\.\d+)+)\.t/
```

CI is failing due to `brew style` saying that this regex should use `//` rather than `%r{}` (since it doesn't contain a forward slash).

I would like to see if the livecheck passes on CI once you fix this.

Reply:
👍